src/handlers/messageHandler.ts

```diff
@@ -15,7 +15,7 @@
 //
 // ________________________________________________________________________________________________
 
-import { type Message, MessageActionRow, MessageButton, NewsChannel, TextChannel, ThreadChannel, SnowflakeUtil, type Snowflake } from "discord.js";
+import { type Message, MessageActionRow, MessageButton, NewsChannel, TextChannel, ThreadChannel, SnowflakeUtil, type Snowflake, MessageAttachment, Permissions } from "discord.js";
```

Comment:
I assume this was VSCode being weird and importing something else.
```suggestion
import { type Message, MessageActionRow, MessageButton, NewsChannel, TextChannel, ThreadChannel, SnowflakeUtil, type Snowflake, Permissions } from "discord.js";
```

Reply:
Good catch!